rotkehlchen/inquirer.py

```diff
@@ -517,25 +518,20 @@ def find_uniswap_v2_lp_price(
             block_identifier='latest',
         )
 
-    def find_curve_pool_price(
-        self,
-        lp_token: EthereumToken,
-    ) -> Optional[Price]:
+    def find_curve_pool_tokens_amount(
+            self,
+            lp_token_address: ChecksumEthAddress,
+    ) -> Optional[Dict[EthereumToken, Tuple[FVal, FVal]]]:
```

Comment:
Since the functionality of this function is no longer related to the price query and is is used externally I believe would be better to have it in a different place. I believe  you can take it to a `utils` file in the curve module and then import it here and in the decoders. This doesn't look like the right place for this method

Reply:
I had in mind that you were not returning prices at all, maybe due to the name of the function but I guess is fine to keep it here